configs/openimages/README.md

```diff
@@ -90,14 +93,14 @@ training/testing by using `tools/misc/get_image_metas.py`.
     │   │   │   ├── class-descriptions-boxable.csv
     │   │   │   ├── oidv6-train-annotations-bbox.scv
     │   │   │   ├── validation-annotations-bbox.csv
-    │   │   │   ├── validation-annotations-human-imagelabels-boxable.csv    # is not necessary
+    │   │   │   ├── validation-annotations-human-imagelabels-boxable.csv    # should set `load_image_level_labels=False` if not use
```

Comment:
1. set load_image_level_label=False in which if not use?
2. Should leave this not in other place.